covigator/references/lineage_annotation.py

```diff
@@ -81,6 +616,8 @@ def _read_constellation_files(self):
             # These calls are incompatible with the scorpio constellation
             incompatible_lineage_calls = set(data["variant"].get("incompatible_lineage_calls", set()))
             pangolin_lineage_list = pangolin_lineage_list - incompatible_lineage_calls
+            lineage_mutations = [self._parse_mutation_sites(x) for x in data["sites"]]
+            lineage_mutations = [mut for x in lineage_mutations for mut in x if mut is not None]
```

Comment:
If you assume there are no `None` in the output list (see other comment), then these two lines can be simplified into one.